hack/make-rules/verify.sh

```diff
@@ -106,10 +107,13 @@ function is-quick {
 function is-explicitly-chosen {
   local name="${1#verify-}"
   name="${name%.*}"
+  index=0
   for e in ${WHAT}; do
```

Comment:
Instead of looping through two distinct objects, just loop through the TARGET_LIST array

```suggestion
  for e in "${TARGET_LIST[@]}"; do
```

Reply:
Good. I will revise it.